docs/Generics/generics.tex

```diff
@@ -4888,7 +4888,7 @@ \section{Primary Archetypes}\label{archetypesubst}
 
 \paragraph{Archetypes are not ``inherited''} There's a potential pitfall worth mentioning. Recall that when generic declarations nest, the inner declaration inherits the generic parameters and requirements of the outer declaration, possibly adding new generic parameters or requirements:
 \begin{Verbatim}
-func myAlgorithm<S: Sequence>(_ seq: S) where S.Element: Comparable {
+func complexAlgorithm<S: Sequence>(_ seq: S) where S.Element: Comparable {
```

Comment:
This function is referred as `complexAlgorithm` on explanation text below.

Reply:
This makes the line in the code display too long, so let's change the other reference to myAlgorithm instead.